lib/filewatch/helper.rb

```diff
@@ -1,5 +1,6 @@
 # code downloaded from Ruby on Rails 4.2.1
 # https://raw.githubusercontent.com/rails/rails/v4.2.1/activesupport/lib/active_support/core_ext/file/atomic.rb
+# change method name to avoid borking active_support and vice versa
```

Comment:
Instead of **monkey_patching** `File`, Why not use an helper module? this would prevent any conflict and would be more clean IMHO.